cmd/graph.go

```diff
@@ -31,7 +31,7 @@ import (
 var (
 	ErrorGoModFileNotFound = errors.New("module file (go.mod) not found")
 
-	licenseRegexp    = regexp.MustCompile(`^(?i)((UN)?LICEN(S|C)E|COPYING).*$`)
+	licenseRegexp    = regexp.MustCompile(`(^|^[a-zA-Z]*/)(?i)((UN)?LICEN(S|C)E|COPYING)[^/]*$`)
```

Comment:
`[a-zA-Z]*` matches 0 or more alphabets, and `/` is the slash for directories. So this should be an attempt at matching any license files in the next level directories

Reply:
> Curious here, what does `(^|^[a-zA-Z]*/)` do here? handling files that could start with XXXLICENSE for example?

This regex avoids filenames with  XXXLICENSE, the `^[a-zA-Z]*/` is meant to avoid directories that start with a `.` like a `.github` folder